### ISSUE TITLE
Make 'Externals' available through editor kitchen sink

### ISSUE THREAD

**Issue by whysthatso** [hedgedoc/hedgedoc#2674]
### Which part of the project should be enhanced?

the kitchen sink of the editor should preferably contain a dropdown with all the supported Externals embed options, ideally inserting a snippet to be completed.

### Is your enhancement request related to a problem? Please describe.

i've just read a comparison to dropbox paper and somebody mentioned easy embeds with either preview snippets a la facebook timeline or proper embeds a la youtube. i remembered that this was a hedgedoc feature but when putting in a youtube link nothing happened. i went back into the feature list of the main site to see that it's necessary to enter a little curly braces snippet.

auto-detecting the necessary embed via the link would be amazing, but the snippets are fine too. but they should be available directly in the editor, so as to have a single place of features. this would dramatically improve the user experience

### Describe the solution you'd like

depending on the feature, a dropdown in the editor kitchen sink might suffice, TOC is another candidate, emojis, todo lists, etc .
basically all the features should be represented.

### Describe alternatives you've considered

a link in the editor to the feature list / feature documentation? but then i would be directed out of context again, so not really preferable.
